packages/firebase_crashlytics/firebase_crashlytics/ios/Classes/FLTFirebaseCrashlyticsPlugin.m

```diff
@@ -5,6 +5,8 @@
 #import "FLTFirebaseCrashlyticsPlugin.h"
 
 #import <Firebase/Firebase.h>
+#import <FirebaseCrashlytics/FIRAnalyticsInterop.h>
+#import <FirebaseCrashlytics/FIRCLSAnalyticsManager.h>
```

Comment:
Causes `fatal error: 'Interop/Analytics/Public/FIRAnalyticsInterop.h' file not found`